tools/test.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# Runs the full test suite.
+# Usage: ./tools/test.py out/Debug
+import os
+import sys
+from check_output_test import check_output_test
+from util import run
+
+
+def main(argv):
+    build_dir = argv[1]
+    os.path.isdir(build_dir)
+    run([os.path.join(build_dir, "test_cc")])
+    run([os.path.join(build_dir, "handlers_test")])
+    check_output_test(os.path.join(build_dir, "deno"))
```

Comment:
On Windows executables have the .exe suffix. Omitting it might work for subprocess.check_output() -- but it causes this assertion to fail:

```
def check_output_test(deno_fn):
    assert os.path.isfile(deno_fn)
```